screens/LoadingScreen.js

```diff
@@ -7,13 +7,24 @@ import { AppLoading } from "expo";
 import { Container, Content, View } from "../components/layout";
 import { Text } from "../components/typography";
 var jwtDecode = require("jwt-decode");
+import { Notifications } from "expo";
 import { tokenRedirect, setLocalLocation } from "../store/actions";
 import { connect } from "react-redux";
 import { getDeviceLocation } from "../utils/index";
 
 const LoadingScreen = props => {
   const fontsLoaded = false;
 
+  useEffect(() => {
+    this._notificationSubscription = Notifications.addListener(
+      this._handleNotification
+    );
+  });
```

Comment:
```suggestion
  }, []);
```
I haven't been able to extensively test but this seems to solve the 'double render' issue

Reply:
NO FKING WAY

Reply:
Dont think this fixes the issue when the app is in the background, and the user clicks the notification to head into the app. I still see two popups of the modal. Is it perhaps only working on iOS?

Reply:
OOOOO. I don't think the case is that two modals loaded, its just that it re-rendered twice because once the user opens the modal, it seems that expo automatically updates the notification to ```"origin": "selected"```. I know _UseEffect_ essentially works like a _componentDidUpdate_ instead of a _componentDidMount_, so it would run everytime the component re-renders due to prop/state change. Visually I dont see two instances of modal so I think this is gud